src/app/public/modules/toast/toast-adapter.service.spec.ts

```diff
@@ -41,16 +41,16 @@ describe('Toast adapter service', () => {
         }
       ]
     });
-    adapter = TestBed.get(SkyToastAdapterService);
+    // adapter = TestBed.get(SkyToastAdapterService);
   });
 
-  it('should append element to body', () => {
-    adapter.appendToBody(undefined);
-    expect(rendererCallCounts.appendCalledCount).toBe(1);
-  });
+  // it('should append element to body', () => {
```

Comment:
?

Reply:
I edited that file to now test the one untested function and removed the tests for the removed methods.